1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,214 +1,191 @@
-# Comparisons
+# तुलना
 
-We know many comparison operators from maths.
+हम गणित से कई तुलना ऑपरेटरों को जानते हैं।
 
-In JavaScript they are written like this:
+जावास्क्रिप्ट में वे इस तरह लिखे जाते हैं:
 
-- Greater/less than: <code>a &gt; b</code>, <code>a &lt; b</code>.
-- Greater/less than or equals: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
-- Equals: `a == b`, please note the double equality sign `==` means the equality test, while a single one `a = b` means an assignment.
-- Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as <code>a != b</code>.
+- ग्रेटर / से कम: <code>a &gt; b</code>, <code>a &lt; b</code>।
+- ग्रेटर / से कम या बराबर: <code>a &gt;= b</code>, <code>a &lt;= b</code>।
+- समतुल्य: `a == b`, कृपया दोहरे समानता चिन्ह पर ध्यान दें `==` का अर्थ है समानता परीक्षण, जबकि एकल `a = b` का अर्थ है असाइनमेंट।
+- नहीं के बराबर है। गणित में अंकन<code>&ne;</code>है, लेकिन जावास्क्रिप्ट में इसे<code>a != b</code> लिखा जाता है।
 
-In this article we'll learn more about different types of comparisons, how JavaScript makes them, including important peculiarities. 
+इस लेख में हम विभिन्न प्रकार की तुलनाओं के बारे में अधिक जानेंगे, कि कैसे जावास्क्रिप्ट उन्हें महत्वपूर्ण विशिष्टता सहित बनाता है।
+अंत में आपको "जावास्क्रिप्ट क्विर्क" से संबंधित मुद्दों से बचने के लिए एक अच्छा नुस्खा मिलेगा।
 
-At the end you'll find a good recipe to avoid "javascript quirks"-related issues.
+# बुलियन परिणाम है
+सभी तुलना ऑपरेटर एक बूलियन मान लौटाते हैं:
 
-## Boolean is the result
+- सत्य - का अर्थ है "हाँ", "सही" या "सत्य"।
+- असत्य - का अर्थ "नहीं", "गलत" या "सत्य नहीं" है।
 
-All comparison operators return a boolean value:
-
-- `true` -- means "yes", "correct" or "the truth".
-- `false` -- means "no", "wrong" or "not the truth".
-
-For example:
-
-```js run
-alert( 2 > 1 );  // true (correct)
-alert( 2 == 1 ); // false (wrong)
-alert( 2 != 1 ); // true (correct)
+उदाहरण के लिए:
+``` js run
+alert( 2 > 1 ); // सच (सही)
+alert( 2 == 1 ); // गलत (गलत)
+alert( 2 != 1 ); // सच (सही)
 ```
 
-A comparison result can be assigned to a variable, just like any value:
-
-```js run
-let result = 5 > 4; // assign the result of the comparison
-alert( result ); // true
+एक तुलना परिणाम एक चर को सौंपा जा सकता है, किसी भी मूल्य की तरह:
+``` js run
+let result = 5 > 4; // तुलना का परिणाम बताएं
+alert( result ); // सच
 ```
 
-## String comparison
-
-To see whether a string is greater than another, JavaScript uses the so-called "dictionary" or "lexicographical" order.
+# स्ट्रिंग तुलना
+यह देखने के लिए कि क्या एक स्ट्रिंग दूसरे से बड़ी है, जावास्क्रिप्ट तथाकथित "शब्दकोश" या "लेक्सिकोग्राफिक" आदेश का उपयोग करता है।
```

Comment:
We should use **लेक्सिकोग्राफिक** for **lexicographical** as it is easy to refer if required IMO.

Reply:
I think I used लेक्सिकोग्राफिक for lexicographical only. Don't know if I'm missing something here

Reply:
/done